source/index.html.md

```diff
@@ -1708,16 +1708,16 @@ By default, all Bank Accounts will be returned.
     {
       "id": "6a7ed958-f1e8-42dc-8c02-3901d7057357",
       "branch_code": "493192",
```

Comment:
Recommend adding in BNZ code such as 020100 and similar for other banks.

Reply:
On it